tests/components/plex/test_config_flow.py

```diff
@@ -0,0 +1,265 @@
+"""Tests for Plex config flow."""
+from unittest.mock import MagicMock, Mock, patch, PropertyMock
+import plexapi.exceptions
+import requests.exceptions
+
+from homeassistant.components.plex import config_flow
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_TOKEN, CONF_URL
+
+from tests.common import mock_coro
+
+
+class MockAvailableServer:
+    """Mock avilable server objects."""
+
+    def __init__(self, name, client_id):
+        """Initialize the object."""
+        self.name = name
+        self.clientIdentifier = client_id
+        self.provides = ["server"]
+
+
+class MockConnection:
+    """Mock a single account resource connection object."""
+
+    def __init__(self):
+        """Initialize the object."""
+        self.httpuri = "http://1.2.3.4:32400"
+        self.uri = "http://4.3.2.1:32400"
+        self.local = True
+
+
+class MockConnections:
+    """Mock a list of resource connections."""
+
+    def __init__(self):
+        """Initialize the object."""
+        self.connections = [MockConnection()]
+
+
+async def test_bad_credentials(hass, aioclient_mock):
```

Comment:
I don't think we're using the aiohttp client mock, since we're not using aiohttp. We can remove the use of this fixture.